### PR TITLE
MBS-7795: Display a map that shows all the places contained in an area

### PR DESCRIPTION
**Pull request by mwiencek**
#386 rebased, with the inline script moved to a separate file, and a few bugs fixed.

I think it's about ready to be merged, but would appreciate others doing a quick test. cc @loujine

### CODE REVIEW

**Comment by loujine** [comment]
\o/ thanks
I'll test it over the weekend, tonight if I can

File: root/static/scripts/area/places-map.js [hunk 1]
```diff
@@ -58,13 +60,22 @@ if (places.length) {
...
-    if (markers.length > 10) {
-      popupText += '<br /> and ' + (markers.length - 10) + ' others';
+
+    if (markers.length > CLUSTER_POPUP_LIMIT) {
+      popupText += '<br /> ';
+      popupText += _.escape(ln(
+        '… and {place_count} other',
+        '… and {place_count} others',
+        CLUSTER_POPUP_LIMIT,
+        {place_count: CLUSTER_POPUP_LIMIT}
```
Comment (loujine) [inline root/static/scripts/area/places-map.js]: Should be {place_count: marker.length - CLUSTER_POPUP_LIMIT}

**Comment by loujine** [comment]
Apart from that, I reviewed the code (at least the part I understood) and tested again locally and everything looks fine

**Comment by mwiencek** [comment]
@loujine Fixed. Thanks again for your work on this. :)

**Pull request merged**